Tutorials/bash/tensorboard-web-app/resources/home/dnanexus/mnist_tensorboard_example.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A simple MNIST classifier which displays summaries in TensorBoard.
+
+This is an unimpressive MNIST model, but it is a good example of using
+tf.name_scope to make a graph legible in the TensorBoard graph explorer, and of
+naming summary tags so that they are grouped meaningfully in TensorBoard.
+
+It demonstrates the functionality of every TensorBoard dashboard.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
+1 for python3 compatibility. Did `from __future__ import unicode_literals` cause issues?

Reply:
nope, everything just worked! This code is straight from the TensorBoard examples (hence the license at the top), so I'm not gonna take credit for this :)

Reply:
I really think we should have gone for 4 examples here. Come on, guys...

Reply:
lol, okay I'll have to get to work on that :)